src/ident.c

```diff
@@ -1,236 +0,0 @@
-/*
- * ProFTPD - FTP server daemon
- * Copyright (c) 1997, 1998 Public Flood Software
- * Copyright (c) 2001-2016 The ProFTPD Project team
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Suite 500, Boston, MA 02110-1335, USA.
- *
- * As a special exemption, Public Flood Software/MacGyver aka Habeeb J. Dihu
- * and other respective copyright holders give permission to link this program
- * with OpenSSL, and distribute the resulting executable, without including
- * the source code for OpenSSL in the source distribution.
- */
-
-/* Ident (RFC1413) protocol support */
-
-#include "conf.h"
-
-static int ident_timeout;
-static pr_netio_stream_t *nstrm = NULL;
-
-static const char *trace_channel = "ident";
-
-static int ident_timeout_cb(CALLBACK_FRAME) {
-  ident_timeout++;
-
-  if (nstrm) {
-    /* Abort the NetIO stream, which will cause netio_poll (and thus
-     * netio_read) to also abort.  This is similar to the way data connects
-     * are aborted.
-     */
-    pr_netio_abort(nstrm);
-  }
-
-  return 0;
-}
-
-char *pr_ident_lookup(pool *p, conn_t *c) {
-  char *ret = "UNKNOWN";
-  pool *tmp_pool = NULL;
-  conn_t *ident_conn = NULL, *ident_io = NULL;
-  char buf[256] = {'\0'}, *tok = NULL, *tmp = NULL;
-  int timerno, i = 0;
-  int ident_port = pr_inet_getservport(p, "ident", "tcp");
-
-  tmp_pool = make_sub_pool(p);
-  ident_timeout = 0;
-  nstrm = NULL;
-
-  if (ident_port == -1) {
-    destroy_pool(tmp_pool);
-    return pstrdup(p, ret);
-  }
-
-  /* Set up our timer before going any further. */
-  timerno = pr_timer_add(PR_TUNABLE_TIMEOUTIDENT, -1, NULL,
-    (callback_t) ident_timeout_cb, "ident lookup");
-  if (timerno <= 0) {
-    destroy_pool(tmp_pool);
-    return pstrdup(p, ret);
-  }
-
-  ident_conn = pr_inet_create_connection(tmp_pool, NULL, -1, c->local_addr,
```

Comment:
I noticed this old file as I was reworking some of the Inet API's connection functions; a `grep` turned up this particular reference to a `pr_inet_create_connection` function, which no longer exists.  And that, in turn, indicates that this file is never compiled.